src/Symfony/Component/Console/Command/Command.php

```diff
@@ -572,6 +572,8 @@ public function getSynopsis($short = false)
      * Add a command usage example.
      *
      * @param string $usage The usage, it'll be prefixed with the command name
+     *
+     * @return Command
```

Comment:
should be `@return self` (as in other places in the code base where fluent interface are in defined)